source/includes/_personnel.md

```diff
@@ -0,0 +1,434 @@
+      "first_name": "Sandra",
+      "last_name": "Smith",
+      "email":"sandra.smith@email.com",
+      "line_manager_id": "456",
+      "role_ids": ["987", "765"]
+    }
+  }
+```
+
+This endpoint allows you to create a personnel. Personnel are division specific, so the personnel will belong to the division that the API key belongs to. You can set the first name, last name and email of the personnel through this endpoint. You can also assign the personnel a line manager using the ID of another personnel, and assign roles to the personnel using the IDs of roles that exist in the account.
+
+### Request
+
+`POST https://api.handshq.com/v1/personnel`
+
+### Allowed Personnel Parameters for create
+All parameters must be nested within `personnel`
+
+Parameter | Format | Required | Description
+--------- | ------ | -------- | -----------
+first_name | String | Yes | First name of the personnel
+last_name | String | Yes | Last name of the personnel
+email | String | No | Email address of the personnel
+line_manager_id | String | No | ID of the line manager of the personnel
+role_ids | Array of strings | No | IDs of roles that exist in the training register, that the personnel holds
+
+### Response
+
+Successful requests will return a json payload of the newly created personnel and a `201` status code
+
+> 201
+
+```json
+  {
+    "data": {
+      "id": "123",
+      "type": "personnel",
+      "attributes": {
+        "first_name": "Sandra",
+        "last_name": "Smith",
+        "email": "sandra.smith@email.com",
+        "archived_at": null,
+        "type": "employee"
+      },
+      "relationships": {
+        "line_manager": {
+          "data": {
+            "id": "456",
+            "type": "line_manager"
+          }
+        },
+        "roles": {
+          "data": [
+            {
+              "id": "987",
+              "type": "role"
+            },
+            {
+              "id": "765",
+              "type": "role"
+            }
+          ]
+        }
+      }
+    }
+  }
+```
+
+## Updating a personnel
+
+```shell
+curl https://api.handshq.com/v1/personnel/[id] \
+  -H "Accept: application/json" \
+  -H "Authorization: bearer [api_token]" \
+  -H "Content-Type: application/json" \
+  --requst PATCH
+  -d '[json_payload]'
+```
+
+> Example Personnel update payload.
+
+```json
+  {
+    "personnel": {
+      "first_name": "Sally",
+      "last_name": "Smith-West",
+      "email":"sally-sw@email.com",
+      "line_manager_id": "567",
+      "role_ids": ["345"]
+    }
+  }
+```
+
+This endpoint allows you to update the first name, last name, email address, line manager and roles of an existing personnel.
+
+### Request
+
+`PATCH https://api.handshq.com/v1/personnel/[id]`
+
+
+### Allowed Personnel Parameters for update
+All parameters must be nested within `personnel`
+
+Parameter | Format | Required | Description
+--------- | ------ | -------- | -----------
+first_name | String | No | First name of the personnel
```

Comment:
@kknevitt , @simhmiller I have set these to required no as they are not required in the context of updating, is that ok? I notice project info still says required yes for update.

Reply:
Ah ok, yeah if anything we can change the product update to have no too 👍 It would be for create but not update, same for personnel (at least for first_name)